recipes/reaktplot/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set version = "0.1.2" %}
+
+package:
+  name: reaktplot
+  version: {{ version }}
+
+source:
+  url: https://github.com/reaktoro/reaktplot/archive/v{{ version }}.tar.gz
+  sha256: 0b2fed13313ebf0934f4d2187a66f1a2c5d6361781944928cc7d87470567a04e
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - catch2 =2
```

Comment:
Looks like a test dependency, perhaps it could be moved to `test/requires` section?